drivers/platform/aducm3029/aducm3029_i2c.c

```diff
@@ -74,17 +75,6 @@ static uint32_t last_bitrate = UNINITIALIZED_BITRATE;
 /** Save the current slave_address to not change it each time */
 static uint8_t last_address = UNINITIALIZED_ADDRESS;
 
-/**
- * @struct i2c_extra
- * @brief I2C platform specific parameters.
- */
-struct aducm_i2c_extra {
-	/** Pointer to the prologue data. */
-	uint8_t *prologue_data;
-	/** Prologue size. */
-	uint8_t prologue_size;
-};
-
```

Comment:
There is no reason to move this to the .h It is only used here. No external code need to access this fields

Reply:
The idea was to keep the code consistent (like any other _extra.h file).